docs/API.md

```diff
@@ -501,6 +501,21 @@ const elem = await $('#elem')
 await expect(elem).toHaveId('elem')
 ```
 
+### toHaveStyle
+
+Checks if an element has specific `CSS` properties.
```

Comment:
@RodBuild, thanks for updating the docs.

Can you explain here if it needs to match each property, or if a single property is enough to when the DOM had more properties? That would make it more clear for the users